components/mpas-ocean/src/Registry.xml

```diff
@@ -1091,175 +1091,175 @@
 		/>
 	</nml_record>
 	<nml_record name="ocean_constants">
-		<nml_option name="config_density0" type="real" default_value="1026.0" units="kg m^{-3}"
+		<nml_option name="config_density0" type="real" default_value="1026.0" units="kg m^-3"
 					description="Density used as a coefficient of the pressure gradient terms, $\rho_0$. This is a constant due to the Boussinesq approximation."
 					possible_values="any positive real, but typically 1000-1035"
 		/>
 	</nml_record>
 	<nml_record name="pressure_gradient" mode="forward">
-		<nml_option name="config_pressure_gradient_type" type="character" default_value="pressure_and_zmid" units="unitless"
+		<nml_option name="config_pressure_gradient_type" type="character" default_value="pressure_and_zmid"
 					description="Form of pressure gradient terms in momentum equation. For most applications, the gradient of pressure and layer mid-depth are appropriate.  For isopycnal coordinates, one may use the gradient of the Montgomery potential.  The sea surface height gradient (ssh_gradient) option is for barotropic, depth-averaged pressure."
 					possible_values="'ssh_gradient', 'pressure_and_zmid' or 'Jacobian_from_density' or 'Jacobian_from_TS' or 'MontgomeryPotential' or 'constant_forced'"
 		/>
-		<nml_option name="config_common_level_weight" type="real" default_value="0.5" units="unitless"
+		<nml_option name="config_common_level_weight" type="real" default_value="0.5"
 					description="The weight between standard Jacobian and weighted Jacobian, $\gamma$."
```

Comment:
Is the use of the latex '$$' allowed? (I noticed this here, but there might be other places that I might have missed)

Reply:
Good call!  Those need to be removed because they just clutter things up now that we're not focusing on the latex documentation.  My guess is that they won't cause errors in the CF-checker though.

@darincomeau, I suppose it's up to you if that's in the scope of this PR or should be a separate one.

Reply:
Thanks for pointing this out @milenaveneziani . But I think trying to remove all latex syntax from attributes like description, which won't impact CF compliance, are more general registry cleanup and outside the scope here, so I think I'd prefer to leave this as is for now rather than trying to do a more complete job of that.

Reply:
sounds good @darincomeau. I just wanted to make sure that it wasn't required.